README.md

```diff
@@ -29,6 +29,11 @@ remoteagent1-dcos112s.example.com
 [agents_public]
 publicagent1-dcos112s.example.com
 
+[agents_windows]
```

Comment:
duplicate

Reply:
Sorry, I didn't get you. There is inventory example and single use of `[agents_windows]` group.

Reply:
sorry, found this duplication now. Removed